docs/Navigation/SetVideoConfig/assets/SetVideoConfigErrorFlow.puml

```diff
@@ -9,8 +9,8 @@ Participant "SDL" as SDL
 Participant "HMI" as HMI
 
 A -> SDL: StartService(serviceType:Video, streamingParams)
```

Comment:
Since this is a protocol message and not an RPC it should be probably be presented a little differently
```suggestion
A -->> SDL: StartService(serviceType:Video, streamingParams)
```